### PR TITLE
Improve imports sanity

### PR DESCRIPTION
**Pull request by antoniovazquezblanco**

### CODE REVIEW

File: include/Suscan/Object.h [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 #include <Suscan/Compat.h>
 
-#include <util/object.h>
+#include <sigutils/util/object.h>
```
Comment (BatchDrake) [inline include/Suscan/Object.h:25]: This is not in sigutils

File: include/Suscan/Config.h [hunk 2]
```diff
@@ -23,7 +23,7 @@
 #include <Suscan/Compat.h>
 #include <Suscan/Object.h>
 #include <vector>
-#include <util/confdb.h>
+#include <sigutils/util/confdb.h>
```
Comment (BatchDrake) [inline include/Suscan/Config.h:26]: This is not in sigutils either

File: include/Suscan/Object.h (hunk 1 above)
Comment (antoniovazquezblanco) [reply include/Suscan/Object.h:25]: Fixed

File: include/Suscan/Config.h (hunk 2 above)
Comment (antoniovazquezblanco) [reply include/Suscan/Config.h:26]: Fixed

**Pull request closed without merging**